modules/rebalance-reference/pages/rebalance-reference.adoc

```diff
@@ -0,0 +1,345 @@
+
+[#standard-fields]
+== Standard Fields
+
+Standard fields are provided for each stage.
+These are as follows:
+
+* `startTime`.
+The point in time at which the stage was started (or possibly, has been determined _not_ to have started).
+Specified as a UTC date/time string.
+
+* `completedTime`.
+The point in time at which the stage was completed (or possibly, has been determined _not_ to have completed).
+Specified as a UTC date/time string.
+
+* `timeTaken`.
+The amount of elapsed time since `startTime`.
+Specified as an integer, in milliseconds.
+The value is `undefined`, if the stage has been determined _not_ to have started.
+
+* `totalProgress`.
+The current, total progress that has been achieved for the stage.
+Specified as a percentage.
+In a completed report, the value is expected to be `100`.
+This is an _optional_ field, which may not appear if not implemented for the stage.
+
+* `perNodeProgress`.
+The current progress that has been achieved for the stage for each node.
+The value is an object, which itself provides a progress figure for each node.
+Each progress figure is specified as a floating-point number between `0.0` and `1.0`.
+In a completed report, the value is expected to be `1`.
+This is an _optional_ field, which may not appear if not implemented for the stage.
+
+* `subStage`.
+Details on any _substages_ that the stage may contain.
+For each _substage_, the standard fields are identical to those for each _stage_.
+This is an _optional_ field, which does not appear, if the stage contains no substages.
+(An example is _delta_recovery_, which is a substage reflected in the `data` output, when Delta Recovery has indeed been performed on one or more nodes running the Data Service.)
+
+* `events`.
+An array of strings, each of which provides information on a significant event.
+The information may be a warning, an error message, or otherwise informational.
+This is an _optional_ field, which does not appear if not implemented for the stage, or if no significant events have occurred.
+
+* `details`.
+Further details of the stage.
+This is an _optional_ field, which is currently implemented for the `data` stage only (information provided below).
+
+* `rebalanceID`.
+A 32-bit string that is the identifier for the rebalance: for example, `"4f64c7dd5a1a452d3bcc3668307d64a6"`.
+
+* `nodesInfo`.
+Information on the nodes of the cluster, in relation to the rebalance.
+The following objects are provided &#8212; each consists of an array of strings, each of which represents a node.
+Where the category has no nodes, the array is empty.
+
+** `active_nodes`.
+The nodes included in the rebalance.
+
+** `keep_nodes`.
+The nodes that were intended to be part of the cluster, following rebalance.
+
+** `eject_nodes`.
+The nodes that were to be ejected by rebalance.
+
+** `delta_nodes`.
+The nodes on which Delta Recovery was to be performed.
+
+** `failed_nodes`.
+The nodes for which rebalance failed.
```

Comment:
These are the nodes that were failed-over prior to rebalance and will be ejected from the cluster on successful rebalance.

Reply:
Done. Thanks.